docs/src/architecture/notes.md

```diff
@@ -5,10 +5,60 @@ A note is a way of transferring assets between accounts. A note consists of a va
     <img src="../diagrams/architecture/note/Note.png">
 </p>
 
-A note consists of:
-* a set of assets stored in a vault.
-* a script which must be executed in a context of some account to claim the assets.
-* a set of inputs which are placed onto the stack before a note's script is executed.
-* a serial number which can be used to break linkability between note and it's nullifier (~ exists when note was already consumed).
+In the above:
+* **Vault** is a set of assets that are stored in note.
+* **Script** that must be executed in a context of some account to claim the assets.
+* **Inputs** which are placed onto the stack before a note's script is executed.
+* **Serial number** is the note's identifier.
```

Comment:
One thing that is probably missing: note metadata. Technically, it is not a part of a note, but for every note we also store note metadata in the Note DB. This metadata includes: user-defined tag (1 field element), sender (1 field element), and number of assets contained in the note (1 field element).

Reply:
They also have metadata, and actually, number of assets is also recorded for them. Might be good to remove that at some point - but I think for now it is fine.